Mathlib/LinearAlgebra/CliffordAlgebra/Even.lean

```diff
@@ -213,15 +214,14 @@ theorem aux_mul (x y : even Q) : aux f (x * y) = aux f x * aux f y := by
   dsimp only
   induction x, x_property using even_induction Q with
   | algebraMap r =>
-    rw [foldr_algebraMap, aux_algebraMap]
+    generalize_proofs at ⊢
+    simp
     exact Algebra.smul_def r _
```

Comment:
Optional style tweak:
```suggestion
    generalize_proofs
    simpa using Algebra.smul_def r _
```
I note that `generalize_proofs` does seem hard to avoid, so this seems like a reasonable application.